src/traits.rs

```diff
@@ -513,3 +517,27 @@ pub trait WideningMul<Rhs = Self>: Sized {
     /// Perform widening multiplication.
     fn widening_mul(&self, rhs: Rhs) -> Self::Output;
 }
+
+/// A representation of an integer optimized for the performance of modular operations.
+pub trait MontyFormLike {
```

Comment:
Maybe just call this `Monty`? I think `*Like` is a little redundant in a trait name.

Reply:
`MontyForm` is already a struct name.

Reply:
`Monty` isn't a struct name, however

Reply:
Ah I see. That seems a little vague to me, but I can live with it.

Reply:
Renamed